implementations/elixir/ockam/ockam_hub/lib/hub/service/provider/token_lease_manager.ex

```diff
@@ -0,0 +1,33 @@
+## TODO: this needs to be moved to ockam_kafka after we create ockam_hub_core
+defmodule Ockam.TokenLeaseManager.Hub.Service.Provider do
+  @moduledoc false
+  @behaviour Ockam.Hub.Service.Provider
+
+  @services [:influxdb_token_manager]
+  @address "token_lease_service"
```

Comment:
Is that generic or specific to influxdb? 
Is there some options to pass to the lease manager to make it influxdb specific?

Reply:
It is generic but, instead of using `@services [:token_manager]` I wanted to list all the possible "implementations" of the token manager, depending on the cloud provider.

Reply:
I's just confusing that the service config is called `influxdb_token_manager`, while the service address is `token_lease_service`, I think address should be more specific then config, because you can extend config with args, but you don't expand address.

Reply:
let's call it just `token_lease_manager`